custom_components/ha_kia_hyundai/services.py

```diff
@@ -24,13 +28,18 @@
 
 _LOGGER = getLogger(__name__)
 
+
 def async_setup_services(hass: HomeAssistant):
     async def async_handle_start_climate(call: ServiceCall):
         coordinator: VehicleCoordinator = _get_coordinator_from_device(hass, call)
         climate = call.data.get(SERVICE_ATTRIBUTE_CLIMATE)
         set_temp = call.data.get(SERVICE_ATTRIBUTE_TEMPERATURE)
         defrost = call.data.get(SERVICE_ATTRIBUTE_DEFROST)
         heating = call.data.get(SERVICE_ATTRIBUTE_HEATING)
+        driver_seat = call.data.get(SERVICE_ATTRIBUTE_DRIVER_SEAT, 0)
```

Comment:
these should default to None instead of 0 for vehicles that don't support seat climate settings